pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/export/tabular/TsvParticipantExporter.java

```diff
@@ -0,0 +1,62 @@
+package org.broadinstitute.dsm.model.elastic.export.tabular;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.List;
+import java.util.Map;
+
+import com.google.common.net.MediaType;
+import org.apache.commons.lang3.StringUtils;
+import spark.Response;
+
+public class TsvParticipantExporter extends TabularParticipantExporter {
+    public static final String DELIMITER = "\t";
+
+    public TsvParticipantExporter(List<ModuleExportConfig> moduleConfigs,
+                                  List<Map<String, String>> participantValueMaps, String fileFormat) {
+        super(moduleConfigs, participantValueMaps, fileFormat);
+    }
+
+
+    public void export(Response response) throws IOException {
+        setResponseHeaders(response);
+
+        PrintWriter writer = response.raw().getWriter();
+        List<String> headerRowValues = getHeaderRow();
+        List<String> subHeaderRowValues = getSubHeaderRow();
+
+        writer.println(getRowString(headerRowValues));
+        writer.println(getRowString(subHeaderRowValues));
+        for (Map<String, String> valueMap : participantValueMaps) {
+            List<String> rowValues = getRowValues(valueMap, headerRowValues);
+            String rowString = getRowString(rowValues);
+            writer.println(rowString);
+        }
+        writer.flush();
```

Comment:
is it chance to have resource leaking after not closing `writer`? Maybe it is better to wrap it in `try-with-resource`.